Description: Use locally served jquery to avoid privacy breach
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2018-09-14
---
This patch header follows DEP-3: https://dep.debian.net/deps/dep3/
--- a/doc/API.htm
+++ b/doc/API.htm
@@ -1804,7 +1804,7 @@
           </div>
     </div>
 
-    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script src="jquery.min.js"></script>
     <script src="index.js"></script>
 </body>
 </html>
--- a/doc/C-style.htm
+++ b/doc/C-style.htm
@@ -1642,7 +1642,7 @@
           </div>
     </div>
 
-    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script src="jquery.min.js"></script>
     <script src="index.js"></script>
 </body>
 </html>
--- a/doc/Commprod.htm
+++ b/doc/Commprod.htm
@@ -312,7 +312,7 @@
           </div>
     </div>
 
-    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script src="jquery.min.js"></script>
     <script src="index.js"></script>
 </body>
 </html>
--- a/doc/DLL.htm
+++ b/doc/DLL.htm
@@ -799,7 +799,7 @@
           </div>
     </div>
 
-    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script src="jquery.min.js"></script>
     <script src="index.js"></script>
 </body>
 </html>
--- a/doc/Deprecated.htm
+++ b/doc/Deprecated.htm
@@ -5796,7 +5796,7 @@
           </div>
     </div>
 
-    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script src="jquery.min.js"></script>
     <script src="index.js"></script>
 </body>
 </html>
--- a/doc/Develop.htm
+++ b/doc/Develop.htm
@@ -4788,7 +4788,7 @@
           </div>
     </div>
 
-    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script src="jquery.min.js"></script>
     <script src="index.js"></script>
 </body>
 </html>
--- a/doc/Devices.htm
+++ b/doc/Devices.htm
@@ -2180,7 +2180,7 @@
           </div>
     </div>
 
-    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script src="jquery.min.js"></script>
     <script src="index.js"></script>
 </body>
 </html>
--- a/doc/Drivers.htm
+++ b/doc/Drivers.htm
@@ -3718,7 +3718,7 @@
           </div>
     </div>
 
-    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script src="jquery.min.js"></script>
     <script src="index.js"></script>
 </body>
 </html>
--- a/doc/Fonts.htm
+++ b/doc/Fonts.htm
@@ -393,7 +393,7 @@
           </div>
     </div>
 
-    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script src="jquery.min.js"></script>
     <script src="index.js"></script>
 </body>
 </html>
--- a/doc/History9.htm
+++ b/doc/History9.htm
@@ -250705,7 +250705,7 @@
           </div>
     </div>
 
-    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script src="jquery.min.js"></script>
     <script src="index.js"></script>
 </body>
 </html>
--- a/doc/Install.htm
+++ b/doc/Install.htm
@@ -463,7 +463,7 @@
           </div>
     </div>
 
-    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script src="jquery.min.js"></script>
     <script src="index.js"></script>
 </body>
 </html>
--- a/doc/Language.htm
+++ b/doc/Language.htm
@@ -2093,7 +2093,7 @@
           </div>
     </div>
 
-    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script src="jquery.min.js"></script>
     <script src="index.js"></script>
 </body>
 </html>
--- a/doc/Lib.htm
+++ b/doc/Lib.htm
@@ -1071,7 +1071,7 @@
           </div>
     </div>
 
-    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script src="jquery.min.js"></script>
     <script src="index.js"></script>
 </body>
 </html>
--- a/doc/Make.htm
+++ b/doc/Make.htm
@@ -1495,7 +1495,7 @@
           </div>
     </div>
 
-    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script src="jquery.min.js"></script>
     <script src="index.js"></script>
 </body>
 </html>
--- a/doc/News.htm
+++ b/doc/News.htm
@@ -245,7 +245,7 @@
           </div>
     </div>
 
-    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script src="jquery.min.js"></script>
     <script src="index.js"></script>
 </body>
 </html>
--- a/doc/Ps-style.htm
+++ b/doc/Ps-style.htm
@@ -572,7 +572,7 @@
           </div>
     </div>
 
-    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script src="jquery.min.js"></script>
     <script src="index.js"></script>
 </body>
 </html>
--- a/doc/Ps2epsi.htm
+++ b/doc/Ps2epsi.htm
@@ -252,7 +252,7 @@
           </div>
     </div>
 
-    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script src="jquery.min.js"></script>
     <script src="index.js"></script>
 </body>
 </html>
--- a/doc/Psfiles.htm
+++ b/doc/Psfiles.htm
@@ -917,7 +917,7 @@
           </div>
     </div>
 
-    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script src="jquery.min.js"></script>
     <script src="index.js"></script>
 </body>
 </html>
--- a/doc/Readme.htm
+++ b/doc/Readme.htm
@@ -602,7 +602,7 @@
           </div>
     </div>
 
-    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script src="jquery.min.js"></script>
     <script src="index.js"></script>
 </body>
 </html>
--- a/doc/Release.htm
+++ b/doc/Release.htm
@@ -868,7 +868,7 @@
           </div>
     </div>
 
-    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script src="jquery.min.js"></script>
     <script src="index.js"></script>
 </body>
 </html>
--- a/doc/SavedPages.htm
+++ b/doc/SavedPages.htm
@@ -340,7 +340,7 @@
           </div>
     </div>
 
-    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script src="jquery.min.js"></script>
     <script src="index.js"></script>
 </body>
 </html>
--- a/doc/Source.htm
+++ b/doc/Source.htm
@@ -489,7 +489,7 @@
           </div>
     </div>
 
-    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script src="jquery.min.js"></script>
     <script src="index.js"></script>
 </body>
 </html>
--- a/doc/Unix-lpr.htm
+++ b/doc/Unix-lpr.htm
@@ -328,7 +328,7 @@
           </div>
     </div>
 
-    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script src="jquery.min.js"></script>
     <script src="index.js"></script>
 </body>
 </html>
--- a/doc/Use.htm
+++ b/doc/Use.htm
@@ -5400,7 +5400,7 @@
           </div>
     </div>
 
-    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script src="jquery.min.js"></script>
     <script src="index.js"></script>
 </body>
 </html>
--- a/doc/VectorDevices.htm
+++ b/doc/VectorDevices.htm
@@ -1517,7 +1517,7 @@
           </div>
     </div>
 
-    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script src="jquery.min.js"></script>
     <script src="index.js"></script>
 </body>
 </html>
--- a/doc/WhatIsGS.htm
+++ b/doc/WhatIsGS.htm
@@ -202,7 +202,7 @@
           </div>
     </div>
 
-    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script src="jquery.min.js"></script>
     <script src="index.js"></script>
 </body>
 </html>
--- a/doc/sample_downscale_device.htm
+++ b/doc/sample_downscale_device.htm
@@ -285,7 +285,7 @@
           </div>
     </div>
 
-    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script src="jquery.min.js"></script>
     <script src="index.js"></script>
 </body>
 </html>
--- a/doc/subclass.htm
+++ b/doc/subclass.htm
@@ -835,7 +835,7 @@
           </div>
     </div>
 
-    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script src="jquery.min.js"></script>
     <script src="index.js"></script>
 </body>
 </html>
--- a/doc/thirdparty.htm
+++ b/doc/thirdparty.htm
@@ -256,7 +256,7 @@
           </div>
     </div>
 
-    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script src="jquery.min.js"></script>
     <script src="index.js"></script>
 </body>
 </html>
--- a/doc/Internal.htm
+++ b/doc/Internal.htm
@@ -141,7 +141,7 @@
           </div>
     </div>
 
-    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script src="jquery.min.js"></script>
     <script src="index.js"></script>
 </body>
 </html>
